lib/dor/update_marc_record_service.rb

```diff
@@ -172,7 +172,7 @@ def get_x2_constituent_info
 
     def released_to_Searchworks
       rel = @druid_obj.released_for
-      rel['Searchworks']['release']
+      rel.blank? || rel['Searchworks'].blank? || rel['Searchworks']['release'].blank? ? false : rel['Searchworks']['release']
```

Comment:
for now, we decided to just make the tags case insensitive --- but the searchworks string itself will still be hardcoded for now

sorry, i forgot to mark this pull request as unnecessary .... if we can merge and deploy the updated dor-services-app and item-release robots, this logic will be gone from here (and be in dor-services-app)

the latest version of this code in dor-services-app makes the whole release tag case insensitive